packages/demo/src/components/examples/InputExamples.tsx

```diff
@@ -343,38 +333,6 @@ const MultilineBoxExamples: React.FunctionComponent = () => (
                 }}
             />
         </Section>
-        <Section
-            level={3}
-            title="A multiline box of inputs with a custom container. Will validate only when the value is the expected one"
-        >
-            <MultilineBoxWithCustomization<IMultilineInputWithMultilineBox>
-                id="secondMultilineBoxExample"
-                data={InitialValues}
-                renderBody={(data: IMultilineInputWithMultilineBoxData, parentProps: {addNewBox: () => void}) =>
-                    _.map(data, (cData: IMultilineSingleBoxProps<IMultilineInputWithMultilineBox>) => (
-                        <div key={cData.id}>
-                            <InputConnected
-                                id={`${cData.id}1`}
-                                classes="mt1 mx1"
-                                defaultValue={cData.props['1']}
-                                onChange={(value: string) => {
-                                    if (value === 'next' && cData.isLast) {
-                                        parentProps.addNewBox();
-                                        return true;
-                                    }
-                                    return false;
-                                }}
-                            />
-                            <InputConnected id={`${cData.id}2`} classes="mt1 mx1" defaultValue={cData.props['2']} />
-                        </div>
-                    ))
-                }
-                defaultProps={{
-                    '1': "Will create another box if you type 'next' here.",
-                    '2': 'But not here.',
-                }}
-            />
-        </Section>

```

Comment:
Why did you remove this example?

Reply:
It probably got caught in the cross fire of me removing ''custom'' examples with the bg-color. I will put it back up :sweat_smile:

Reply:
I'm just not a fan of opening the door to custom bg-color to replicate the original example... Thought? 
![image](https://user-images.githubusercontent.com/63734941/107519521-5e796f00-6b7e-11eb-87bb-aba3deaaf023.png)